packages/backend/app/Class/Quiz.ts

```diff
@@ -212,30 +202,41 @@ export default class Quiz extends Room {
       // bad answer
       this.playerWrongAnswer(player);
     }
-  };
+  }
 
-  resetPlayers = () => {
+  private resetPlayers() {
     this.players.forEach((player) => player.reset());
-  };
+  }
 
-  resetRoom = () => {
+  private resetRoom() {
     this.roundsCounter = 0;
     this.emitRoundCounter();
-  };
+  }
 
-  resetPlayersForNewRound = (): void => {
+  private resetPlayersForNewRound(): void {
     this.players.forEach((player) => player.resetForNewRound());
-  };
-
-  setPlayersFind = (find: boolean): void => {
-    this.players.forEach((player) => player.setFind(find));
-  };
+  }
 
-  startGame = (socket: Socket) => {
+  public startGame(socket: Socket) {
     this.emitStatusToSocket(socket.id);
     if (this.status === RoomStatus.Waiting && this.players.length >= 1) {
       this.event.emit(GameEvent.Start);
     }
     socket.on(GameEvent.Guess, (guess) => this.playerGuess(socket.id, guess));
-  };
+  }
+
+  private async setRandomRounds() {
```

Comment:
nit: I would rather rename this something like `pullRandomRounds()`. or `pullRandomRounds()`
We're not creating or modifying rounds, we're just pulling new ones. 
```suggestion
  private async pullRandomRounds() {
```